app/jobs/send_purge_warnings_job.rb

```diff
@@ -8,15 +8,12 @@ def perform(*_args)
     sent = []
     not_sent = []
 
-    # Get admin users
-    recipients = User.where(role: [Roles::ADMIN, Roles::SUPER_USER])
+    # Get admin users but skip USA admins and locked users
+    recipients = User.where(role: [Roles::ADMIN, Roles::SUPER_USER], locked_at: nil).where.not(jurisdiction_id: Jurisdiction.find_by(name: 'USA').id)
```

Comment:
This makes me think we should try to make the concept of "USA" jurisdiction a bit less hard coded. There are a few places where we can improve things, but since we're here let's start with the admin panel.

Please add a static method to the Jurisdiction model that does this bit, i.e.:

```
def self.root
  Jurisdiction.find_by(name: 'USA')
end
```

And then we can change this to be:

```
recipients = User.where(role: [Roles::ADMIN, Roles::SUPER_USER], locked_at: nil).where.not(jurisdiction: Jurisdiction.root)
```

This will also let us fix some other area where this is hard coded!

Reply:
Worth noting there's built-in `ancestry` functionality for this in some ways, now that you mention it.
You can do `Jurisdiction.roots.first` to get this as well.
https://github.com/stefankroes/ancestry